debian/control

```diff
@@ -69,6 +70,7 @@ Depends: ${misc:Depends},
          apg,
          colord (>= 0.1.30),
          desktop-file-utils,
+         libeos-parental-controls-0,
```

Comment:
This should not be necessary because it should automatically be a part of `${shlibs:Depends}`, I think.